core/posting.go

```diff
@@ -1,14 +1,19 @@
 package core
 
+import "regexp"
+
+// Posting struct
 type Posting struct {
-	Source      string `json:"source"`
-	Destination string `json:"destination"`
-	Amount      int64  `json:"amount"`
-	Asset       string `json:"asset"`
+	Source      string `json:"source" validate:"required,source"`
+	Destination string `json:"destination" validate:"required,destination"`
+	Amount      int64  `json:"amount" validate:"required,min=2"`
```

Comment:
min=2 is by spec?
I think about 1 BTC.

Reply:
Yes sure! I was about to send the commit. I forgot to revert the configuration while testing the validation. 
Thanks